FridgeExercise/Refrigerator.cs

```diff
@@ -0,0 +1,249 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace FridgeExercise
+{
+	internal class Refrigerator
+	{
+		public int RefrigeratorID { get; set; }
+		private int CounterID = 1;
+		public string Model { get; set; }
+		public string Color { get; set; }
+		public int NumberOfShelves { get; set; }
+		public List<Shelf> Shelves { get; set; }
+
+
+		public Refrigerator(string model, string color, int numberOfShelves) {
+			RefrigeratorID = CounterID;
+			CounterID++;
+			Model = model;
+			Color = color;
+			NumberOfShelves = numberOfShelves;
+			Shelves = new List<Shelf>();
+
+		}
+
+		public string ToString(Refrigerator refrigerator)
+		{
+			return "Refrigerator: Model:  " + refrigerator.Model + "  Color: " + refrigerator.Color + "  NumberOfShelves:  "+ refrigerator.NumberOfShelves + "";
+		}
+
+		public double SpaceIsLeftInTheFridge(List<Shelf> shelves)
```

Comment:
```suggestion
		public double GetAvailableSpace(List<Shelf> shelves)
```

Try looking for a consice call to action